chart/templates/_helpers.yaml

```diff
@@ -47,7 +47,7 @@
   - name: AIRFLOW__CELERY__BROKER_URL
     valueFrom:
       secretKeyRef:
-        name: {{ .Release.Name }}-broker-url
+        name: {{ default (printf "%s-broker-url" .Release.Name) .Values.redis.brokerURLSecretName }}
```

Comment:
having dynamically changing values for secretRef is a bit weird definitely (instead of values changing depending on context)

Reply:
Yeah, I don't find this cart particularly clean ; but what I've done seems pretty close to the existing stuff in the chart.